hack/lib/util/environment.sh

```diff
@@ -145,12 +145,14 @@ function os::util::environment::setup_tmpdir_vars() {
     HOME="${FAKE_HOME_DIR}"
     export HOME
 
-    # ensure that the directories are clean
-    for target in $( ${USE_SUDO:+sudo} findmnt --output TARGET --list ); do
-        if [[ "${target}" == "${BASETMPDIR}"* ]]; then
-            ${USE_SUDO:+sudo} umount "${target}"
-        fi
-    done
+		if [[ "$(uname)" != "Darwin" ]]; then
```

Comment:
i don't think we ever mount something inside OSX as the openshift server won't start standalone on OSX (without using oc cluster up). i think it is reasonable to opt-out this on osx.